app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -18,9 +18,16 @@ <h5 class="sidebar-title">Editing project</h5>
           </div>
         </li>
         <li>
-          <span class="label">Add/Remove Scenes</span>
-          <button class="btn btn-primary fixedwidth" type="button" ui-sref="projects.edit.scenes">
+          <span class="label">Scene Management</span>
+          <a class="btn btn-primary fixedwidth" ui-sref="projects.edit.scenes">
```

Comment:
Agreed. My thought is that if we are using a `ui-sref`, it should be a `a` so that it ends up being a real link, which is nice to have wherever possible.